jobs/env/ci-kubernetes-kubemark-gce-scale.env

```diff
@@ -1,31 +1,17 @@
-### job-env
-
-# Increase throughput in Kubemark master components.
-KUBEMARK_MASTER_COMPONENTS_QPS_LIMITS=--kube-api-qps=100 --kube-api-burst=100
-# Increase limit for inflight requests in apiserver (we do the same in production clusters).
-TEST_CLUSTER_MAX_REQUESTS_INFLIGHT=--max-requests-inflight=1500 --max-mutating-requests-inflight=500
-# Increase throughput in Load test.
-LOAD_TEST_THROUGHPUT=10
-# Override defaults to be independent from GCE defaults and set kubemark parameters
-# We need 11 so that we won't hit max-pods limit (set to 100). TODO: do it in a nicer way.
+### cluster-env
 NUM_NODES=80
-MASTER_SIZE=n1-standard-4
 # Note: can fit about ~9 hollow nodes per core so NUM_NODES x
 # cores_per_node should be set accordingly.
 NODE_SIZE=n1-standard-8
-# Increase disk size to check if that helps for etcd latency.
-MASTER_DISK_SIZE=100GB
-# =========================================
-# Configuration we are targetting in 1.6
-STORAGE_MEDIA_TYPE=application/vnd.kubernetes.protobuf
-# The kubemark scripts build a Docker image
-JENKINS_ENABLE_DOCKER_IN_DOCKER=y
 
-# TODO: revert after running experiments.
-EVENT_PD=true
-# TODO: Reduce this once we have log rotation in Kubemark.
-KUBEMARK_MASTER_ROOT_DISK_SIZE=100GB
+### kubernetes-env
```

Comment:
Why did you remove MASTER_SIZE, MASTER_DISK_SIZE?

Reply:
They'd be auto-calculated based on NUM_NODES.
Ref: https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/config-test.sh#L32-L34